src/Components/pages/Book.js

```diff
@@ -8,9 +9,15 @@ const propTypes = {};
 const defaultProps = {};
 
 const Book = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(displayBookFunction());
+  }, []);
   const bookList = useSelector((state) => state.bookReducer);
-  return bookList.map((item) => (
-    <SingleBook key={item.id} item={item} />
+
+  return bookList.map((item, index) => (
```

Comment:
- [ ] let’s not disable linter rules as per Microverse guidelines. Linters are here to help you write clean code

Reply:
Solved thank you